Detections/AWSCloudTrail/AWS_ConsoleLogonWithoutMFA.yaml

```diff
@@ -1,6 +1,9 @@
 ﻿id: d25b1998-a592-4bc5-8a3a-92b39eedb1bc
 name: Login to AWS Management Console without MFA.
-description: Multi-Factor Authentication (MFA) helps you to prevent credential compromise. This query looks for login to AWS Management Console without MFA. This is done by looking at eventName of ‘ConsoleLogin and looking in the AdditionalEventData field to check if MFA was used and then confirming from the ResponseElements field that the login was a success.
+description: |
```

Comment:
invalid ymal file, here and in another files the description change from one line to multiline.

on using '|' or '|-' the lines should be indent. 
for examle:
description: |
    indent
    indent

you can use online yaml validetor to check it.

Reply:
fixed